releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,5 +1,16 @@
 ## Microsoft Extensions Azure App Configuration
```

Comment:
BTW, can we change the title to the actual package name and add a link of the package under the title, for example
## Microsoft.Extensions.Configuration.AzureAppConfiguration
[Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)